### PR TITLE
Define missing ui:order

### LINKED ISSUE #107: Field ordering must be consistent.
The ordering of the fields isn’t consistent. I suggest this order: 
- “Enabled” checkbox,
- blocklist filters,
- blocklist metadata (name, bug, descriptions), 
- read-only fields.

Always show min version before max version, and show them next to each other (there’s a case where there’s another input between them).

### PR DESCRIPTION
**Pull request by Natim**
Fixes #107.

### CODE REVIEW

**Comment by Natim** [comment]
Please review this one really carefully, it is really error prone since from gfx, plugins and addons versionRange doesn't mean the same thing.

File: amo-blocklist.json [hunk 1]
```diff
@@ -199,6 +199,25 @@
             "why": {"ui:widget": "textarea"},
             "who": {"ui:widget": "textarea"},
             "created": {"ui:readonly": true}
+          },
+          "versionRange": {
```
Comment (n1k0) [inline amo-blocklist.json]: If this is the same definition across collections, shouldn't we try using a reusable [definition reference](https://github.com/mozilla-services/react-jsonschema-form#schema-definitions-and-references)?

Comment (Natim) [inline amo-blocklist.json]: Not is is not exactly the same.

Comment (Natim) [inline amo-blocklist.json]: Also I am not sure we can use definition reference in the uiSchema object, can we?

Comment (n1k0) [inline amo-blocklist.json]: > Not is is not exactly the same.
  
Sigh ;)

> Also I am not sure we can use definition reference in the uiSchema object, can we?

Not with JSON indeed, though that could be possible with JavaScript... which is not an option here :/

**Pull request merged**